tests/src/DependencyInjection/ContainerTest.php

```diff
@@ -385,6 +385,25 @@ public function test_initialized() {
     $this->assertTrue($this->container->initialized('late.service'), 'Late service is initialized after it was gotten.');
   }
 
+  /**
+   * Camelizes a string.
+   *
+   * @covers ::camelize
+   */
+  public function test_camelize() {
+    $result = $this->container->camelize('service_container');
+    $this->assertEquals('ServiceContainer', $result, 'The strings are the same.');
+  }
+
+  /**
+   * Un-camelizes a string.
+   *
+   * @covers ::underscore
```

Comment:
I would like to use data providers for that and test a little more scenarios.

Note:

The same data provider can be used here for both functions.